synopackage_dotnet.model/synopackage_dotnet.model.csproj

```diff
@@ -65,7 +65,7 @@
     <PackageReference Include="HolopaMir.ExpressMapper" Version="2.0.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.Compilers" Version="3.11.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="3.11.0">
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.1.0">
```

Comment:
Probably need to go back to version 3.x, as 4.x is targeting .NET 6 and actions fails